### DISCUSSION TITLE
Columns not rendering in .pdf

### DISCUSSION THREAD

**Discussion by giorgiopaulon** [quarto-dev/quarto-cli#1669] [Q&A]
Hi all! 

I am working on a Quarto template that can render many features across several output types, i.e., .pdf, .docx and .HTML. One of the features that I am not able to export correctly is a list of items in a column environment. This shows correctly in the HTML but not in the .pdf generated by Quarto (as seen in the pictures below). 

I attach below a reproducible example. Am I missing something?

```
---
title: "Columns"
format: 
  # html:
  #   self-contained: true
  pdf:
    documentclass: article
---

:::: {.columns}

::: {.column width="50%"}
-   [ ] Item 1
-   [ ] Item 2
-   [x] Item 3
:::

::: {.column width="50%"}
-   [x] Item on the right
:::

::::
```
<img width="444" alt="Screen Shot 2022-08-03 at 7 01 05 PM" src="https://user-images.githubusercontent.com/90702867/182735688-7432647e-3680-4d2b-a5bd-0b3c50d5189e.png">

<img width="547" alt="Screen Shot 2022-08-03 at 7 01 42 PM" src="https://user-images.githubusercontent.com/90702867/182735694-116975e1-7343-4f7b-9319-c5f9a26348d9.png">

**Comment by stibu81** [comment]
This is probably much too late for you, but maybe it will help someone else. It seems that the .columns-syntax does not work with pdf articles. (It works with beamer, though.) I found [here](https://stackoverflow.com/a/77306118/4303162) that the following works for pdf articles:

```
---
title: "Columns"
format: 
  # html:
  #   self-contained: true
  pdf:
    documentclass: article
---

:::: {layout="[1, 1]"}

::: {}
-   [ ] Item 1
-   [ ] Item 2
-   [x] Item 3
:::

::: {}
-   [x] Item on the right
:::

::::
```

<img width="1002" height="534" alt="columns" src="https://github.com/user-attachments/assets/18f80321-3956-4f7f-a876-f6e174cbc976" />

This also worked for html and docx.

**Reply by mcanouil** [reply]
@stibu81

> [!IMPORTANT]
> Accessibility: To improve accessibility, please add alternative text to your screenshots. This helps all users, including those using screen readers, to understand the context of the images. A brief description can make a big difference! See [Good Alt Text, Bad Alt Text — Making Your Content Perceivable](https://www.wcag.com/blog/good-alt-text-bad-alt-text-making-your-content-perceivable/).

> [!IMPORTANT]
> Accessibility: For better accessibility, please ensure link text is descriptive and meaningful rather than using generic terms such as "here". This helps all users understand the link's destination. See [Writing meaningful link text (WCAG)](https://www.wcag.com/blog/writing-meaningful-link-text/).

**Reply by mcanouil** [reply]
@stibu81 Better to not resurrect 4 years old dead thread (also marked as "stale"), as it does not really help and trigger noise notifications to maintainers and participants.

**Comment by mcanouil** [comment]
The layout feature is documented on <quarto.org>:

- https://quarto.org/docs/authoring/figures.html#block-layout
- https://quarto.org/docs/authoring/figures.html#complex-layouts
- https://quarto.org/docs/authoring/figures.html#figure-panels

`.column` is a Pandoc feature for presentations:

- https://quarto.org/docs/presentations/index.html#multiple-columns
